### ISSUE TITLE
Improve Accessibility of site

### ISSUE THREAD

**Issue by LikkanChung** [CSSUoB/cssuob.github.io#147]
Axe/WAVE Accessibility Testing currently flags various issues across the site..

This should be improved to make it more friendly for people with accessibility needs.

**Comment by MattyTheHacker** [comment]
Lighthouse recommendations:

![Screenshot 2022-07-12 145455](https://user-images.githubusercontent.com/18513864/178507248-cc69261d-dcd3-40dd-8065-086381b318fe.png)
